packages/auto-pr-labeler/src/assignLabels.ts

```diff
@@ -1,12 +1,56 @@
 import * as core from '@actions/core';
 
+import { espressoStaff } from './espressoStaff';
 import { PR_REVIEW_DECISION, PR_STATE } from './constants';
 import { repoLabels } from './labels';
-import { addLabelsMutation, removeLabelsMutation } from './mutations';
+import { addAssigneesMutation, addLabelsMutation, removeLabelsMutation } from './mutations';
 
-import type { ID, IssueConnection, LabelList, LabelsQueryResponse, PullRequest, RepoName } from './types';
+import type {
+	AssigneesQueryResponse,
+	ID,
+	IssueConnection,
+	LabelList,
+	LabelsQueryResponse,
+	PullRequest,
+	RepoName,
+} from './types';
 import type { GraphQlQueryResponse } from '@octokit/graphql/dist-types/types';
 
+/**
+ * adds the supplied assignees to the specified Issue or Pull Request
+ *
+ * @param assignableId ID
+ * @param assigneeIds [ID]
+ * @returns Promise<GraphQlQueryResponse<AssigneesQueryResponse>>
+ */
+const addAssignees = async (
+	assignableId: ID,
+	assigneeIds: Array<ID>
+): Promise<GraphQlQueryResponse<AssigneesQueryResponse>> => {
+	try {
+		// eslint-disable-next-line no-console
+		console.log('%c addAssignees', 'color: HotPink;', { assignableId, assigneeIds });
```

Comment:
Any reason to always log? Perhaps only do it in the event of error?

Reply:
just so i can see the output when the action runs